Tasks/NodeToolV0/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 176,
+        "Minor": 178,
```

Comment:
Could you please bump version to the current sprint (179)?

Reply:
Done